integration/metrics_test.go

```diff
@@ -218,7 +218,7 @@ log_fuse_operations = true
 
 			sh.X("soci", "image", "rpull", "--soci-index-digest", indexDigest, imgInfo.ref)
 			// this command may fail due to fuse operation failure, use XLog to avoid crashing shell
-			sh.XLog("ctr", "run", "--rm", "--snapshotter=soci", imgInfo.ref, "test", "echo", "hi")
+			sh.XLog("nerdctl", "run", "--name", "test", "--pull", "never", "--net", "none", "--rm", "--snapshotter", "soci", imgInfo.ref, "echo", "hi")
```

Comment:
Should we extract parts of these common commands into a shared location?

eg:

```go
// Commands
runCmd := []string{"nerdctl"", "run", "--pull", "never", "--net", "none", "--snapshotter", "soci"}
pullCmd := []string{"soci", "image", "rpull"}

// pull an image
sh.X(append(pullCmd, "testImage")...)
// run a container
sh.X(append(runCmd, "testImage", "echo", "hello")...)

```

Reply:
I like the idea — trying to refactor this was a massive pain for that reason. Could be something shared around the package in like utils.go or something. Should that be a separate PR, though?

Reply:
I don’t think that refactor makes this PR any larger or more complex, so it’s probably fine to do it now.

Reply:
In that case I'll just stick to changing the run commands. I can do a separate PR after this is merged consolidating all of the other commands.